### DISCUSSION TITLE
Ethernet / Power Over Ethernet

### DISCUSSION THREAD

**Discussion by Wapata** [alf45tar/PedalinoMini#448] [Ideas]
Hi,
Just wondering if I'm the only one to think about POE Power Over Ethernet ? Or even simple Ethernet...
I mean, wifi is cool, but in a near future where a lot of midi device can talk over network, isn't a wired solution kind of better in live ?
Powering stuff with a unique single cable sound cooler than thinking about every battery or every power cable adapters.

What do you think ?

**Comment by alf45tar** [comment]
Hi Wapata,
I think the ESP32 board with ethernet and POE are not so common and the number of available GPIO is no more than 6.
It is out of scope of PedalinoMini at the moment.

**Reply by Wapata** [reply]
Mmmm it's about the boards themself.
After the Arduino micro used as bridge to usb-midi, I was wondering why I didn't find any daughter boards with this add-on (and thought that I was simply bad at searching). 
Thank you for your answer 🙏

**Reply by mknerr** [reply]
I don't believe there's any reason you couldn't follow the POE specifications and tap power from the Ethernet jack to a proper voltage regulator to power the Pedalino, or perhaps just use an adapter to split the power to a barrel connector (again, making sure your voltage regulator can handle the voltage your router is putting out). https://a.co/d/fWsks7K

**Reply by Wapata** [reply]
Yes for the power but too bad for all the midi-ethernet capabilities. 
But in the end, a dedicated for music ethernet/wifi router should do the job. I will see that if I need it for real :-)

**Comment by Wapata** [comment]
Hi !
I've found something !
You use https://github.com/BlokasLabs/USBMIDI as a bridge BUT, it seem that https://github.com/FortySevenEffects/arduino_midi_library#other-transport-mechanisms is more powerful (like... a lot if you just look at the "github stars" at the top of the pages...

As mentioned here https://github.com/alf45tar/PedalinoMini#usb-midi-using-arduino-pro-micro , in the end the arduino is "just a bridge" so I will give a try to that solution if i find a small couple of ethernet shield/arduino.

That said, I hope this info will be usefull for someone. See you ! ;-)
